src/backend/utils/__mocks__/indexer.js

```diff
@@ -20,8 +27,22 @@ const deletePost = (postId) => {
   return Promise.resolve();
 };
 
-const search = (keyword = '') => {
-  const filtered = db.values.filter((value) => value.text.includes(keyword));
+const mockCreateFieldsFromFilter = (filter) => {
```

Comment:
This is fine to duplicate for now.  Rename it to `createFieldsFromFilter` like the other.